packages/node_modules/pouchdb-core/src/adapter.js

```diff
@@ -220,34 +220,29 @@ AbstractPouchDB.prototype.put = adapterFun('put', function (doc, opts, cb) {
   }
   const self = this;
   if (opts.force && doc._rev) {
-    transformForceOptionToNewEditsOption(function () {
-      putDoc(function (err) {
-        const result = err ? null : {ok: true, id: doc._id, rev: doc._rev};
-        cb(err, result);
-      });
+    transformForceOptionToNewEditsOption();
+    putDoc(function (err) {
+      const result = err ? null : {ok: true, id: doc._id, rev: doc._rev};
+      cb(err, result);
     });
   } else {
     putDoc(cb);
   }
 
-  function transformForceOptionToNewEditsOption(next) {
-    self.get(doc._id, {revs: true, rev: doc._rev}, function (err, obj) {
-      /* istanbul ignore if */
-      if (err) {
-        return next(err);
-      }
-      const revisions = obj._revisions;
-      const nRevNum = revisions.start + 1;
-      const newRevId = uuid(32, 16).toLowerCase();
-      revisions.start = nRevNum;
-      revisions.ids.unshift(newRevId);
+  function transformForceOptionToNewEditsOption() {
+    const parts = doc._rev.split('-');
+    const oldRevId = parts[1];
+    const oldRevNum = parseInt(parts[0]);
 
-      doc._revisions = revisions;
-      doc._rev = nRevNum + '-' + newRevId;
-      opts.new_edits = false;
+    const newRevNum = oldRevNum + 1;
+    const newRevId = uuid(32, 16).toLowerCase();
```

Comment:
No, it is not. pouchdb generate new rev id just like this when `new_edits=true`. see here https://github.com/pouchdb/pouchdb/blob/63f039cb0c79671a0bcab7e2cf2afc7be297e23c/packages/node_modules/pouchdb-adapter-utils/src/parseDoc.js#L93